default.nix

```diff
@@ -1,7 +1,10 @@
+{ nurpkgs ? import <nixpkgs> {} # For nixpkgs dependencies used by NUR itself
+  # Dependencies to call NUR repos with
+, pkgs ? throw "NUR call didn't receive a pkgs argument, but the evaluation requires it."
```

Comment:
This is new, I think this should be okay, there's no need to provide an implicit pkgs

Reply:
Also coincides with requiring a `pkgs` before this PR, as it was `pkgs.callPackage (fetchGit ...)`

Reply:
This change is great because it will help users when using NUR wrong.

Reply:
I can add a note that says it could be an error on either side.